viz_scripts/scaffolding.py

```diff
@@ -202,8 +202,8 @@ def mapping_color_labels(dynamic_labels, dic_re, dic_pur):
     if len(dynamic_labels) > 0:
         mode_values = list(mapping_labels(dynamic_labels, "MODE").values()) if "MODE" in dynamic_labels else []
         replaced_mode_values = list(mapping_labels(dynamic_labels, "REPLACED_MODE").values()) if "REPLACED_MODE" in dynamic_labels else []
```

Comment:
Nit, future fix: shouldn't we add the "Other", similar to https://github.com/Abby-Wheelis/em-public-dashboard/commit/75ff38672fa5ed9c16c76a09796faeb9f0d39c31 for mode and replaced mode as well. We don't currently have any deployments with custom values that don't include "Other", but we might!

Reply:
We definitely should make sure each list has "Other" to prevent issues

Reply:
Ah, Other is already added to the modes when we combine replaced and mode -- `combined_mode_values = mode_values + replaced_mode_values + ['Other']` and `combined_mode_values` is what is used to make the color mapping